src/dsp_tools/xmllib/__init__.py

```diff
@@ -13,7 +13,6 @@
 from .models.dsp_base_resources import VideoSegmentResource as VideoSegmentResource
 from .models.resource import Resource as Resource
 from .models.root import XMLRoot as XMLRoot
-from .value_checkers import find_geometry_problem as find_geometry_problem
```

Comment:
Since they don't write the geometry by hand anymore, we can remove this function.